main.tf

```diff
@@ -17,6 +17,12 @@ locals {
     local.create_security_group ? local.created_security_group.id : coalesce(var.target_security_group_id[0],
     "var.target_security_group_id contains null value. Omit value if you want this module to create a security group.")
   ) : null
+
+  # Setting `create_before_destroy` on the security group rules forces create_before_destroy behavior
+  # on the security group, so we have to disable it on the rules if disabled on the security group.
+  rule_create_before_destroy = var.create_before_destroy && var.rule_create_before_destroy
+  # We also have to make it clear to Terraform that the CBD rules will never reference the DBC security group
```

Comment:
```suggestion
  # We also have to make it clear to Terraform that the CBD rules will never reference the CBD security group
```

Reply:
is `DBC` correct and means "Destroy before create"?
If yes, maybe update the comment to mention that, and also it's not clear at the first glance how the line below  makes it clear to Terraform that the CBD rules will never reference the xxx security group

Reply:
Comments updated